python/src/trezorlib/cli/cardano.py

```diff
@@ -75,6 +87,11 @@ def sign_tx(client, file, signing_mode, protocol_magic, network_id, testnet):
         for withdrawal in transaction.get("withdrawals", ())
     ]
     auxiliary_data = cardano.parse_auxiliary_data(transaction.get("auxiliary_data"))
+    mint = cardano.parse_mint(transaction.get("mint", ()))
+    additional_witness_requests = [
+        cardano.parse_additional_witness_requests(p)
```

Comment:
Should it be `request` instead of `requests`?

Reply:
https://github.com/vacuumlabs/trezor-firmware/pull/66/commits/5464dda6bd13dee7dc3c63c36c6024753e8f3c46#diff-aa355fcce844deffafbd696c1d414eb1c0198b0af2726710c7b8d8f7fda577cdR538